SurfaceDuoSamples/src/main/res/layout/duo_samples_fragment_first.xml

```diff
@@ -0,0 +1,315 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.Licensed under the MIT License.
+  -->
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@android:color/white"
+    android:orientation="vertical"
+    tools:context=".DuoSamplesListFragment">
+
+    <ScrollView
+        android:id="@+id/scroll"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_margin="@dimen/duo_sample_small_margin"
+        android:layout_marginEnd="0dp">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/duo_sample_medium_margin"
+                android:layout_marginBottom="25dp"
+                android:text="@string/hello_to_the_app"
+                android:textColor="@color/dual_view_black"
+                android:textSize="20sp"
+                android:textStyle="bold" />
+
+            <androidx.cardview.widget.CardView
```

Comment:
We have many cardview items that makes this xml layout suuuuuper long, we maybe can create independent layouts for each cardview and then use `include` tag.

Reply:
Or maybe we should add a RecyclerView with items for each sample. Probably, in the future we'll have multiple samples.

Reply:
Also, we should try to use ConstraintLayout instead of oldest layouts

Reply:
We will add a recycler view at a later stage